### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by TravelingTechGuy** [enzoampil/fastquant#195]
#### Problem description

`ImportError: cannot import name 'get_stock_data'`

Full Traceback:
```bash
Traceback (most recent call last):
  File "fastquant.py", line 1, in <module>
    from fastquant import get_stock_data
  File "/c/Users/test/fastquant.py", line 1, in <module>
    from fastquant import get_stock_data
ImportError: cannot import name 'get_stock_data'
```
#### Example

```python
from fastquant import get_stock_data
jfc = get_stock_data("JFC", "2018-01-01", "2019-01-01")
print(df.head())

```

#### Expected behavior

Supposed to run.

#### Environment

-  platform (Windows 10 WSL2 running Ubuntu 18.04.5 LTS):
- Python version (3.6.9)
-  fastquant version (latest)
-  installation method (pip  20.2.2)

**Comment by enzoampil** [comment]
Hi @TravelingTechGuy , thanks for using fastquant! From what directory are you running python?

Can you tell me what you get when you run the code below:
```
import os
os. getcwd()
```

**Comment by TravelingTechGuy** [comment]
Running this code in the same file's directory yields the directory name I'm in `"/c/Users/test/"`.

What were you suspecting/hoping it'd show?

**Comment by enzoampil** [comment]
Right, I'm wondering why there's a `fastquant.py` in your current directory. Do you mind navigating to a different directory and running your script again there?

**Comment by TravelingTechGuy** [comment]
I tried navigating to the parent directory - same result.

I then tried using `venv` (virtual environment), installed fastquant in it - and am still getting the same error.

I feel that I'm doing something wrong, but I'm not sure what 😟 (I'm a Node developer - I touch Python only when I have to).

**Comment by TravelingTechGuy** [comment]
Ok, figured it out. I called my file `fastquant.py`. Upon running it in a VENV, I noticed the last line: 

```bash
ImportError: cannot import name 'get_stock_data' from partially initialized module 'fastquant' (most likely due to a circular import) (C:\Users\Test\fastquant.py)
```

I realized that what threw this off was that the name of my file was the same as the name of the library! Upon changing the file name to `test.py` all is running well.

I'm not sure if there's something that needs to be done in the library to protect from circular import, or just a warning to not call your file `fastquant.py` (or is it something experienced Python developers know not do already). But I'm OK with closing this issue.

Thanks for all your efforts and time!
